### ISSUE TITLE
wrong handling of internal record structs 

### ISSUE THREAD

**Issue by theKBro** [danielpalme/ReportGenerator#685]
internal record structs are handled incorrectly

```
namespace ClassLibrary1;

public class DummyMix
{
    private readonly record struct DummyInternalRecord(string Type, string Name, string Attribute);

    private readonly DummyInternalRecord _internalRecord;
    
    public DummyMix(string name)
    {
        _internalRecord = new DummyInternalRecord("string", name, "");
    }

    public string Name => _internalRecord.Name;

}

```
- getting coverage with dotcover  
- converting with reportgenerator  
leads to multiple  
```
<method name="Name" signature="()" line-rate="1" branch-rate="1" complexity="NaN">
    <lines>
      <line number="7" hits="1" branch="false" />
    </lines>
  </method>
```

with missing DummyInternalRecord context

for the full dummy project see
[ClassLibrary1.zip](https://github.com/user-attachments/files/16495881/ClassLibrary1.zip)

if I can help to inspect this further please don't hesitate to contact me.
thanks in advance

**Comment by theKBro** [comment]
this leads to jenkins recordCoverage errors e.g. `Found unhandled java.lang.IllegalArgumentException exception:
There is already the same child [METHOD] Name() <0> with the name Name() in [CLASS]`

possibly related with #630

**Comment by danielpalme** [comment]
Will have a look within the next 3-4 weeks.
Probably I won't be able to do anything about this problem, but we will see.

**Comment by theKBro** [comment]
I filed a PR, would be great if you could have a look at it.

**Comment by danielpalme** [comment]
Yes. I will have a look as soon as possible.
But it will take some time!

**Comment by danielpalme** [comment]
This is related to #630 (_Cobertura_ format instead of _dotCover_). 

**Some history**:
When I started to implement _ReportGenerator_ only _HTML_ format was available.
I decided to display nested classes and structs within the parent class.

This was a reasonable approach, since HTML is for human users.

Later _Cobertura_ was added as an output format. Here is makes more sense to treat nested types separately.


**Bad news**
- I won't merge your PR, because it will change the default handling of nested classes

**Good news**
- In #630 I added a "raw mode" for _Cobertura_ files. I now did the same for _dotCover_ (see 4444f14f0b574d4deda7921e3ab6892c823474c7) 
- You need a [PRO license](https://reportgenerator.io/pro) to use the new feature.<br/>I can give you a license for your efforts, just send me a short message (https://reportgenerator.io/contact)

The new feature will be available in version 5.3.9 (I will release as soon as all open issues are resolved)

The setting (`settings:rawMode=true`) disables that coverage data of nested or compiler generated classes is included in the parent class.

Limitations: 
- Raw mode is currently only supported for _Cobertura_ and _dotCover_ files.
- If you want _HTML_ report and _Cobertura_ output, you should execute _ReportGenerator_ twice:
  - **Cobertura**: `settings:rawMode=true`
  - **HTML**: `settings:rawMode=false`
 
Documentation of the new feature:
https://reportgenerator.io/features#rawmode
https://reportgenerator.io/pro
https://github.com/danielpalme/ReportGenerator/wiki/Settings#other-settings
